src/Octoshift/Commands/WaitForMigration/WaitForMigrationCommandHandler.cs

```diff
@@ -75,9 +75,11 @@ private async Task WaitForOrgMigration(string migrationId, GithubApi githubApi)
 
     private async Task WaitForRepositoryMigration(string migrationId, GithubApi githubApi)
     {
+        _log.LogInformation($"Waiting for migration (ID: {migrationId}) to finish...");
+
         var (state, repositoryName, failureReason, migrationLogUrl) = await githubApi.GetMigration(migrationId);
 
-        _log.LogInformation($"Waiting for {repositoryName} migration (ID: {migrationId}) to finish...");
+        _log.LogInformation($"Waiting for migration of repository (name: {repositoryName}) to finish...");
```

Comment:
Small :pinching_hand: phrasing nit :grin: 

```suggestion
        _log.LogInformation($"Waiting for migration of repository {repositoryName} to finish...");
```